alibuild_helpers/init.py

```diff
@@ -17,7 +17,15 @@ def parsePackagesDefinition(pkgname):
   return [ dict(zip(["name","ver"], y.split("@")[0:2]))
            for y in [ x+"@" for x in list(filter(lambda y: y, pkgname.split(","))) ] ]
 
-def doInit(setdir, configDir, pkgname, referenceSources, dist, defaults, dryRun):
```

Comment:
I'd leave it as it was, we're redefining variables for no reason.

Reply:
Do you insist on reverting this change? I had to add a variable to the argument list and thought it is getting too long. So I followed the example of `doBuild`, because all arguments are a member of the `args` dictionary already. This has also an impact on the calls in the unit test files.

Reply:
Say, in any case it does not pertain to the topic of this PR (neither does adding Fedora to the list of supported platforms!) I would revert it for now (we need to get the nice feature you've developed quickly available), and if I implemented it I would not duplicate variables like this (there's plenty of `blah = args.blah`, why not accessing `args.blah` directly then?)

Reply:
(Just leave the Fedora part there, it's fine...)

Reply:
> why not accessing `args.blah` directly then?

Done :P